web/src/PredictImpactMode.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  import type { Feature } from "geojson";
+  import { GeoJSON, LineLayer } from "svelte-maplibre";
+  import { Popup } from "svelte-utils/map";
+  import { SplitComponent } from "svelte-utils/top_bar_layout";
+  import BackButton from "./BackButton.svelte";
+  import { layerId, Link } from "./common";
+  import { ModalFilterLayer } from "./layers";
+  import { backend, mode } from "./stores";
+
+  // Based partly on https://colorbrewer2.org/#type=diverging&scheme=RdYlGn&n=5
+  // The middle color white doesn't matter; the source data will filter out unchanged roads
+  let divergingScale = ["#1a9641", "#a6d96a", "white", "#fdae61", "#d7191c"];
+
+  let data = $backend!.predictImpact();
+
+  let minRoadWidth = 3;
+  let maxRoadWidth = 10;
+
+  function pickRoad(f: Feature) {
+    $mode = { mode: "impact-detail", road: f };
+  }
+</script>
+
+<SplitComponent>
+  <div slot="top">
+    <nav aria-label="breadcrumb">
+      <ul>
+        <li>
+          <Link on:click={() => ($mode = { mode: "title", firstLoad: false })}>
+            Choose project
+          </Link>
+        </li>
+        <li>
+          <Link on:click={() => ($mode = { mode: "network" })}>
+            Pick neighbourhood
+          </Link>
+        </li>
+        <li>Predict impact mode</li>
```

Comment:
```suggestion
        <li>Predict impact</li>
```

"mode" feels redundant in the UI

Reply:
Done